src/modules/ModuleEarnShared/RoiCalculator/index.vue

```diff
@@ -343,6 +343,9 @@ const detailsList = computed(() => {
 }
 
 .input-alt-units {
+  overflow: hidden;
+  width: 100%;
+  white-space: nowrap;
```

Comment:
Maybe it's better to use `text-overflow: ellipsis` as well?

Reply:
I don't think so, cause it will break ability to scroll it to the right side to the end of number